packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/configure-packages/react-native/installoktaauthsdk.md

```diff
@@ -0,0 +1,24 @@
+## Install dependency
+
+```bash
+npm install @okta/okta-react-native
+```
+
+## Configure the SDK
+
+Call the `createConfig` function to create a configured client when app is initialized. You can put it in `index.js` or `App.js`. Resolves to `true` if it successfully configures a client.
```

Comment:
adding "the" in front of app

...a configured client when the app is initialized.